core/clause_6_introduction.adoc

```diff
@@ -2,33 +2,37 @@
 
 === Motivation
 
-JavaScript Object Notation (JSON) is a popular encoding format for geospatial data. The light weight, simple syntax, and clear human and machine readability of JSON as well as universal support in all modern programming languages appeals to developers. GeoJSON has become a very popular encoding and is supported in most deployments of APIs implementing OGC API Features. However, GeoJSON has limitations that prevent or limit its use in some cases, including:
+JavaScript Object Notation (JSON) is a popular encoding format for geospatial data. The light weight, simple syntax, and clear human and machine readability of JSON as well as universal support in all modern programming languages appeals to developers. The GeoJSON format as defined in https://datatracker.ietf.org/doc/html/rfc7946[IETF RFC-7946] is a very popular encoding.  GeoJSON is supported by many commercial and open source geospatial techonology product and libraries.  GeoJSON is also supported in most deployments of APIs implementing the OGC API Features Standard. However, GeoJSON has limitations that prevent or limit its use in many cases, including:
 
-* WGS 84 is the only coordinate reference system that is supported;
+* WGS 84 is the only allowed Coordinate Reference System (CRS);
+* No support for metadata so such properties as the provenance of the data cannot be specified;
+* No styling or symbology information is specified;
 * Geometries are restricted to points, curves and surfaces with linear interpolation; and
 * No general capability is available to specify the schema of a feature, its type or its temporal characteristics.
 
+NOTE: PAV Comment.  I added Carl's comments about provenance metadata and styling information but I am not sure they are relevant here.  There is nothing preventing someone from adding additional members to GeoJSON to support these requirements.  They would simply be ignored by GeoJSON clients.  Also, I think these requirements are out-of-scope for the JSON-FG SWG.  The Metadata DWG (?) and the Styling/Symbology SWG (?) would need to consider extending GeoJSON with additional members to accommodate these requirements.
```

Comment:
Meeting 2022-03-07: Yes, these use cases are out-of-scope for JSON-FG and we should not add them in this document. We will limit the list to the topics that JSON-FG addresses. The key point is that we are not prohibiting others to add such information to a GeoJSON/JSON-FG document. @pvretano will remove the two bullets again.

Reply:
Resolved by 97ebaf7